app/helpers/application_helper.rb

```diff
@@ -3,4 +3,15 @@ def show_avatar
     avatar = current_user.profile.avatar.present? ? current_user.profile.avatar : "http://lorempixel.com/26/26/"
     image_tag(avatar, class: 'avatar')
   end
-end
+
+  def markdown(text)
+    options = { autolink: true, highlight: true, fenced_code_blocks: true }
+    renderer = Redcarpet::Render::HTML.new(hard_wrap: true)
+    markdown = Redcarpet::Markdown.new(renderer, options)
+    markdown.render(text).html_safe
+  end
+
+  def make_body_attr
+    {controller: params[:controller].gsub('/', ' ').titleize.gsub(' ', '.'), action: action_name.capitalize}
```

Comment:
Se for o caso, acho que ja rola colocar o código ai direto ao invés de definir um helper só para isso ja que isso só vai ser usado nesse lugar.

Tem como tu testar @joridos ?